src/plugins/vis_augmenter/common/types.ts

```diff
@@ -9,19 +9,25 @@ export enum VisLayerTypes {
   PointInTimeEvents = 'PointInTimeEvents',
 }
 
+export type PluginResourceType = string;
+
+export interface PluginResource {
+  type: PluginResourceType;
+  id: string;
+  name: string;
+  urlPath: string;
```

Comment:
What would be an example of this urlPath look like?

Reply:
It would be something like `anomaly-detection-dashboards#/detectors/<detector-id>/configurations`. The base path would come from core services. 

Full example:
```
<EuiLink href={`${getCore().http.basePath.prepend(`${urlPath}`)}`}>{name}</EuiLink>
```

Reply:
@ohltyler Is there an updated design doc for these data models? Mostly just want to make sure our docs stay in sync with these changes that come out of the actual implementation. We also just may want to have a placeholder issue to remind us to document example usage (you may have already created one).

Reply:
Is it enough for `urlPath` to be a static string? In your example, it seems like maybe you'd want to be able to pass something like a `detector-id`?